sklearn/pipeline.py

```diff
@@ -305,6 +305,12 @@ def predict(self, X):
             Data to predict on. Must fulfill input requirements of first step
             of the pipeline.
 
+        **predict_params : dict of string -> object
+            Parameters passed to the final ``predict`` in the pipeline. Note
+            that uncertainties that are generated by the transformations
```

Comment:
"while this may be used to return uncertainties from some models with return_std or return_cov, "